packages/venia-ui/lib/queries/removeItem.graphql

```diff
@@ -0,0 +1,13 @@
+mutation remoteItem($cartId: String!, $itemId: Int!) {
+    removeItemFromCart(input: { cart_id: $cartId, cart_item_id: $itemId }) {
+        cart {
```

Comment:
Seems unnecessary to make the server fetch this data if we just ignore it. Did you intend to pass to cartDetails.receive and remove the getCartDetails action call?

Reply:
You cannot ignore it, AFAICT graphQl requires you have something.

However that's not necessarily a bad thing, as you suggest. We can probably do away with the `getCartDetails` call so long as this data provides whatever we need.

Reply:
You got half of my point, but not worth another cycle. IF response data is required and we're not doing anything with it right now, might be best just to request `cart { id }` or something more minimal so the backend isn't doing all this extra work for nothing.

Reply:
Fair. Whomever does `getCartDetails` can just go through all this :)